src/controllers/user_controller.cr

```diff
@@ -14,4 +14,14 @@ class UserController < ApplicationController
       redirect_to "/"
     end
   end
+
+  def remove_handle
```

Comment:
We need a CSRF protection on this action too, otherwise someone can simply points a user to ```/users/remove_handle?_method=put``` and remove this signed in user's handle (a deauth-like attack)